frontend/src/pages/LandingPage.js

```diff
@@ -1,33 +1,36 @@
 // SPDX-License-Identifier: MIT
 // SPDX-FileCopyrightText: 2021 Christoph Ehm <christoph.ehmendoerfer@campus.tu-berlin.de>
+// SPDX-FileCopyrightText: 2021 Dominic Heil <d.heil@campus.tu-berlin.de>
 
-import * as React from "react";
+import React from "react";
```

Comment:
What is the difference between both? Is this change needed?

Reply:
I think my IDE changed that.

Reply:
But also not really an issue...